pkg/server/commands/write_authzmodel.go

```diff
@@ -53,6 +57,10 @@ func (w *WriteAuthorizationModelCommand) Execute(ctx context.Context, req *openf
 		Conditions:      req.GetConditions(),
 	}
 
+	if w.rejectConditions && model.Conditions != nil {
```

Comment:
We'll probably want to guard the case where `model.Conditions` is an empty slice as well (e.g. `[]*openfgav1.RelationshipCondition{}`.

If the conditions are unspecified (e.g. nil) or empty then it should be fine. Tooling may normalize no conditions to an empty slice.